libsel4camkes/docs/marshall_macros.md

```diff
@@ -0,0 +1,12 @@
+<!--
+     Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+
+     SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Marshall macros
+
+These are a collection of macros intended to be used with the marshalling and
+unmarshalling process of CAmkES RPC functional calls. The RPC-related CAmkES
+connectors are built with the help of these macros and outside of the
+connectors, there does not seem to be a use case for these macros.
```

Comment:
I think rather than say, 'there does not seem to be a use case for these macros' (which implies to me they should be removed), say, 'These macros are used by the CAmkES tools to build RPC-connectors; they not intended as a public interface' or something